oem/ibm/host-bmc/host_lamp_test.cpp

```diff
@@ -70,6 +70,11 @@ bool HostLampTest::asserted(bool value)
                 "Get Effecter ID failed, effecter = 0");
         }
     }
+    else
+    {
```

Comment:
instead of new else block we can set Asserted property at line 74 itself

Reply:
modified as per comment.